### DISCUSSION TITLE
please help with installing a single-phase solid state relay

### DISCUSSION THREAD

**Discussion by Dragotim** [Arksine/moonraker#424] [Q&A]
good evening, can you help with installing a single-phase solid state relay to a printer on a clipper (rasberry zero used this code ( tried many different [power printer]
type: gpio
pin: gpiochip0/gpio26 # uses pin 26 on gpiochip0
off_when_shutdown: True
initial_state: off

[power my_pin]
type: clipper_device
object_name: output_pin my_pin gives error on startup Failed to load power device [power printer] Error parsing option (pin) from section [power printer]

Unparsed config option 'pin: gpiochip0/gpio26 # uses pin 26 on gpiochip0' detected in section [power printer]. This may be an option no longer available or could be the result of a module that failed to load. In the future this will result in a startup error.

**Comment by Arksine** [comment]
Assuming you are attempting to toggle gpio26 of the Pi, I would retry the first method.  Inline comments are not supported in `moonraker.conf`, so remove `# uses pin 26 on gpiochip0` and see if it works.

The `klipper_device` type is specifically for pins configured in Klipper, it doesn't sound like that is what you are after.
